### ISSUE TITLE
Make the barcharts clickable

### ISSUE THREAD

**Issue by Avsecz** [kipoi/website#56]

**Comment by zupan** [comment]
@krrome you are right. DataTables library allows entering the pre-search string that is used to filter down the result set when initializing table on page load.

Just the JS logic of reading from and updating the URL is needed and then we have a solution allowing to link to arbitrary subset and also automatically generating reflecting URL for any search query.

**Comment by krrome** [comment]
Perhaps this is useful, but maybe an overkill as we don't need full matting of a get-request: https://stackoverflow.com/questions/8486099/how-do-i-parse-a-url-query-parameters-in-javascript

**Comment by zupan** [comment]
True. There are simpler snippets made exactly for this features that we can addapt: https://github.com/wet-boew/wet-boew/issues/6622#issuecomment-265833271
